doc/release-notes.rst

```diff
@@ -2,6 +2,127 @@
  Release Notes
 ===============
 
+v12.2.2 Luminous
+================
+
+This is the second bugfix release of Luminous v12.2.x long term stable
+release series. It contains a range of bug fixes and a few features
+across CephFS, RBD & RGW. We recommend all the users of 12.2.x series
+update.
+
```

Comment:
There are also several major items backported to this release:

- Standby ceph-mgr daemons now redirect requests to the active messenger, easing configuration of tools and humans that access the web dashboard, restful API, or other ceph-mgr module services.
- The prometheus module has several significant updates and improvements.
- The new balancer module enables automatic optimization of CRUSH weights to balance data across the cluster.
- The ceph-volume tool has been updated to include support for BlueStore as well as FileStore.  The only major missing ceph-volume feature is dm-crypt support.

Reply:
thanks